### PR TITLE
Add Python 3.8.10150.0

### PR DESCRIPTION
**Pull request by Croydon**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19773)

### CODE REVIEW

File: manifests/p/Python/Python/3/3.8.1015.0/Python.Python.3.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+LicenseUrl: https://docs.python.org/3/license.html
+Moniker: python
+Tags:
+- python
+- python3
+- programming language
+- script
+- glue language
+PackageUrl: https://www.python.org
+Publisher: Python
```
Comment (ItzLevvie) [inline manifests/p/Python/Python/3/3.8.1015.0/Python.Python.3.locale.en-US.yaml:14]:
```suggestion
Publisher: Python Software Foundation
```

File: manifests/p/Python/Python/3/3.8.1015.0/Python.Python.3.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+PackageName: Python 3.8.10
```
Comment (ItzLevvie) [inline manifests/p/Python/Python/3/3.8.1015.0/Python.Python.3.locale.en-US.yaml:15]:
```suggestion
PackageName: Python 3
```

File: manifests/p/Python/Python/3/3.8.1015.0/Python.Python.3.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+PackageIdentifier: Python.Python.3
+PackageVersion: 3.8.1015.0
```
Comment (ItzLevvie) [inline manifests/p/Python/Python/3/3.8.1015.0/Python.Python.3.locale.en-US.yaml:3]:
```suggestion
PackageVersion: 3.8.10150.0
```

File: manifests/p/Python/Python/3/3.8.1015.0/Python.Python.3.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: Python.Python.3
+PackageVersion: 3.8.1015.0
```
Comment (ItzLevvie) [inline manifests/p/Python/Python/3/3.8.1015.0/Python.Python.3.installer.yaml:3]:
```suggestion
PackageVersion: 3.8.10150.0
```

File: manifests/p/Python/Python/3/3.8.1015.0/Python.Python.3.yaml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+PackageIdentifier: Python.Python.3
+PackageVersion: 3.8.1015.0
```
Comment (ItzLevvie) [inline manifests/p/Python/Python/3/3.8.1015.0/Python.Python.3.yaml:3]:
```suggestion
PackageVersion: 3.8.10150.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll have to update the version folder too.

**Comment by Croydon** [comment]
I just double-checked and it works locally. Both validating and installing. No clue what is going on in CI

**Comment by Croydon** [comment]
Bot says it worked, CI checks say it's failed. What now?

**Review by ItzLevvie — APPROVED** [review]

**Comment by ItzLevvie** [comment]
cc @palenshus @denelon: "Azure-Pipeline-Passed" label needs to be added to this PR.

**Review by denelon — APPROVED** [review]

**Pull request merged**